contrib/automatic_container_release/run_checker.sh.tpl

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# Generates a docker run command to run a automatic container release config
+# validator released as a Docker image. Argument list:
+# docker_path- Path to the docker executable.
+# spec_file_path- Absolute path to the YAML spec file to validate.
+# spec_file_container_path- The path to copy the spec file to in the docker
+#                           container.
+# image_name- The docker image for the checker that will be run.
+# cmd_args- Arguments to pass to the checker.
+
+function guess_runfiles() {
+    if [ -d ${BASH_SOURCE[0]}.runfiles ]; then
+        # Runfiles are adjacent to the current script.
+        echo "$( cd ${BASH_SOURCE[0]}.runfiles && pwd )"
+    else
+        # The current script is within some other script's runfiles.
+        mydir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+        echo $mydir | sed -e 's|\(.*\.runfiles\)/.*|\1|'
+    fi
+}
+
+RUNFILES="${PYTHON_RUNFILES:-$(guess_runfiles)}"
+
```

Comment:
Do a check for the docker_tool_path here as in other templates: https://github.com/bazelbuild/rules_docker/blob/master/docker/util/extract.sh.tpl#L6

Reply:
Done